res/values/strings.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="mtm_accept_cert">Accept Unknown Certificate?</string>
+    <string name="mtm_security_risk">Warning: Potential Security Risk</string>
```

Comment:
I see some issues with this name change: 
- it's not strictly related to the PR at hand
- the dialog title is getting longer, not shorter
- I'm really not sure if it is adding or removing clarity - the old title was a clear, albeit technical, problem statement; the new one is just scary words
- changing a string means I need to get all the translations updated

Reply:
Users who have no clue what certificates are can also use apps using MTM. If they see a dialog that they don't understand there is a risk that they will choose a random option or a wrong option. With the new term (that is basically a copy of what Firefox displays that should make it familiar to some users) non technical users will be alerted that something bad can happen if they accept it.

Reply:
Currently, there is nothing in the dialog indicating that there can be a risk in accepting it.

Reply:
Alright, you got me convinced. Will we still have a useful string about what the security risk is at the top of the dialog in all code paths?

Reply:
Yes. I will attach screenshots in a separate comment in a moment.